types/editorjs__list/editorjs__list-tests.ts

```diff
@@ -0,0 +1,11 @@
+import List from '@editorjs/list';
```

Comment:
```suggestion
import List = require('@editorjs/list');
```
old pattern for UMD when used on DT
see other comment